plugins/providers/metabase/client.go

```diff
@@ -166,11 +199,145 @@ func (c *client) GetCollections() ([]*Collection, error) {
 	if _, err := c.do(req, &collection); err != nil {
 		return nil, err
 	}
-
+	c.logger.Info(fmt.Sprintf("Fetch total %d collections from request: %v", len(collection), req.URL))
```

Comment:
With `/salt/log`, we can utilise its [kv pair logging](https://github.com/odpf/salt/blob/main/log/logger.go#L13), like this

```suggestion
	c.logger.Info(fmt.Sprintf("Fetch collections from request, "total", len(collection), "url", req.URL))
```